src/main/java/ro/vladutit/Don/t/forget/v2/controller/CategoryController.java

```diff
@@ -41,6 +41,8 @@ public void deleteItem(@PathVariable (value = "id") Long id) {
     @GetMapping("/categories/{name}")
     @ResponseBody
     public boolean checkCategoryName(@PathVariable(value = "name") String name) {
-        return categoryService.getCategoryByName(name);
+        boolean res = categoryService.getCategoryByName(name);
+        System.out.println("I am in and response is: " + res);
```

Comment:
we don't need this anymore, right? please in the future don't forget to remove any debug related code

Reply:
I first wait for confirmation that everything is ok, so that I don't write them again, but delete the remnants at the end when I receive the approval. That's what I thought.